### DISCUSSION TITLE
Discussion on GitHub for each episode?

### DISCUSSION THREAD

**Discussion by ojacques** [Tech-Debt-Burndown/TDBwebsite#5] [Ideas]
It would be nice to get a discussion topic going on GitHub for each TDB episode.

**Comment by cpswan** [comment]
Maybe we can start with yours?

It's great to have guests here to chat :)

**Reply by ojacques** [reply]
A while ago, I was looking to add [utteranc.es](https://utteranc.es/) to my own blog. Then I just found out about this brand new fork - [Giscus](https://giscus.vercel.app/), which does the same - but with GitHub discussions - instead of (ab)using GitHub issues. 
Should be straightforward to add it to the theme even (and also issue a PR upstream).
